Update interface (#9)
OriumNetwork/EIPs: EIPS/eip-7432.md

```solidity
/// @title ERC-7432 Non-Fungible Token Roles
/// @dev See https://eips.ethereum.org/EIPS/eip-7432
/// Note: the ERC-165 identifier for this interface is 0x565ccd2b.
interface IERC7432 /* is ERC165 */ {

 /// @notice Emitted when a role is granted.

>     /// @param _grantor The role creator.

Reviewer: The user never creates or deletes roles. Users grant (or assign) and revoke only.

Please fix this on all occurrences

```suggestion
    /// @notice Emitted when a role is granted.
    /// @param _grantor The user assigning the role.
```

    /// @param _role The role identifier.
    /// @param _tokenAddress The token address.
    /// @param _tokenId The token identifier.
    /// @param _grantee The user that receives the role assignment.
    /// @param _expirationDate The expiration date of the role assignment.
    /// @param _data Any additional data about the role assignment.
    event RoleGranted(
        address indexed _grantor,
        bytes32 indexed _role,
        address indexed _tokenAddress,
        uint256 _tokenId,
        address _grantee,
        uint64 _expirationDate,
        bytes _data
    );

    /// @notice Emitted when a role is revoked.
    /// @param _revoker The role revoker.
    /// @param _role The role identifier.
    /// @param _tokenAddress The token address.
    /// @param _tokenId The token identifier.
    /// @param _grantee The user that receives the role revocation.
    event RoleRevoked(
        address indexed _revoker,
        bytes32 indexed _role,
        address indexed _tokenAddress,
        uint256 _tokenId,
        address _grantee

>     );

Reviewer: The indexed parameters should be `_role`, `_tokenAddress` and `_tokenId`. `_tokenAddress` alone is useless, as it's not enough to identify an NFT

>     /// @notice Emitted when an operator is approved to grant a role to another user.

Reviewer: ```suggestion
    /// @notice Emitted when an account is approved to manage roles on behalf of another user.
```

    /// @param _grantor The role creator.
    /// @param _operator The user that can grant the role.
    /// @param _tokenAddress The token address.
    /// @param _isApproved The approval status.
    event RoleApprovalForAll(
        address indexed _grantor,
        address indexed _operator,
        address indexed _tokenAddress,
        bool _isApproved
    );

>     /// @notice Emitted when an operator is approved to grant a role to another user.

Reviewer: ```suggestion
    /// @notice Emitted when an account is approved to manage roles of an NFT on behalf of another user.
```
